vpr/src/route/rr_graph2.cpp

```diff
@@ -2561,20 +2381,20 @@ static void label_incoming_wires(const int chan_num,
                                  const int max_len,
                                  const enum Direction dir,
                                  const int max_chan_width,
-                                 std::vector<int>* labels_ptr,
+                                 std::vector<RRNodeId>* labels_ptr,
                                  int* num_incoming_wires,
                                  int* num_ending_wires) {
     /* Labels the incoming wires on that side (seg_num, chan_num, direction).
      * The returned array maps a track # to a label: array[0] = <the new hash value/label for track 0>,
      * the labels 0,1,2,.. identify consecutive incoming wires that have sblock (passing wires with sblock and ending wires) */
 
-    std::vector<int>& labels = *labels_ptr;
+    std::vector<RRNodeId>& labels = *labels_ptr;
     int itrack, start, end, num_passing, num_ending, pass;
     bool sblock_exists, is_endpoint;
 
     /* Alloc the list of labels for the tracks */
     labels.resize(max_chan_width);
-    std::fill(labels.begin(), labels.end(), UN_SET);
```

Comment:
Same comment on UN_SET -- seems like it should be RRNodeId::Invalid() if this is actually storing an rr_node index.  And if it isn't, we should change the type of the vector back to int.